_images/project-data/migrating-legacy-designer2-proc.adoc

```diff
@@ -0,0 +1,46 @@
+[id='migrating-from-legacy-designer-proc-legacy-designer-proc']
+
+= Migrating projects from the deprecated legacy process designer
+
+The legacy process designer in {CENTRAL} is deprecated in {PRODUCT} {PRODUCT_VERSION_LONG}. It will be removed in a future {PRODUCT} release. The legacy process designer will not receive any new enhancements or features. Existing users of the legacy process designer should start migrating projects to the new designer and all new projects should be created using the new process designer.
+
+Only business processes that contain nodes types that are supported in the new designer can be migrated. More node types will be supported in future versions of {PRODUCT}.
+
+[NOTE]
+====
+The legacy process designer is disabled by default in {CENTRAL}. To enable the legacy process designer in {CENTRAL}, in the upper-right corner of the window click *Settings* -> *Roles*, select a role from the left panel, click *Editors -> Select Editor -> (Legacy) Process Designer -> Read*, and then click *Save* to save the changes.
+====
+
+.Prerequisites
+* You have an existing project that was created in the legacy process designer.
+
+.Procedure
+. In {CENTRAL}, click *Menu* -> *Design* -> *Projects*.
+. Click on the project you want to migrate, for example *{PROJECT}*.
+. Click *Ok* to open the project's asset list.
+. Click on the project's *Business Process* asset to open it in the legacy process designer.
+. Click *Migrate* -> *Migrate Diagram*.
+
++
+.Migration confirmation message
+image::project-data/migrate-message.png[Migration confirmation message]
+
+. Select *Yes* or *No* to confirm whether or not to save your changes.
+
++
+.Save diagram changes confirmation
+image::project-data/save-changes-migration.png[Save diagram changes confirmation]
+
+If the migration is successful, the business process opens in the new process designer and the business process name's extension changes from `*.bpmn2` to `*.bpmn`.
+
+If the migration is unsuccessful due to an unsupported node type, {Central} will provide the following error message:
+
+.Migration failure message
+image::project-data/migrate-fail.png[Migration failure message]
+
+*Note to dev:* I will add the following note to our existing docs and then link them to the procedures in this document so that we don't bulk up all impacted docs with this process.
+
+[NOTE]
+====
+The legacy process designer in {CENTRAL} is deprecated in {PRODUCT} {PRODUCT_VERSION_LONG}. It will be removed in a future {PRODUCT} release. The legacy process designer will not receive any new enhancements or features. Existing users of the legacy process designer are advised to migrate projects to the new designer. For information about migrating projects to the new designer, see xref:migrating-from-legacy-designer-proc[_Migrating projects from the deprecated legacy process designer_].
```

Comment:
similar comment as above here, instead of saying 
"to migrate projects to the new designer" we can probably say " to migrate their processes to the new designer"

Reply:
Done as suggested. 

Here is the revised version: http://file.rdu.redhat.com/~mhaglund/BXMSDOC-4093NEW/#migrating-from-legacy-designer-proc